src/pages/article/Article.jsx

```diff
@@ -1,9 +1,15 @@
-const Article = () => {
-    return ( 
-        <div className="article-page">
-            Article
-        </div>
-     );
-}
- 
-export default Article;
\ No newline at end of file
+const Article = ({ article }) => {
+  return (
```

Comment:
You need to fetch the data for the card, inside here - using useParams - grabbing the article id from the url